stan/math/prim/fun/binomial_coefficient_log.hpp

```diff
@@ -1,10 +1,16 @@
 #ifndef STAN_MATH_PRIM_FUN_BINOMIAL_COEFFICIENT_LOG_HPP
 #define STAN_MATH_PRIM_FUN_BINOMIAL_COEFFICIENT_LOG_HPP
 
+#include <boost/math/constants/constants.hpp>
```

Comment:
What constant do you need from boost? I must have missed it, but I didn't see any. If you need the header, move it after the Stan headers.

Reply:
My bad, I used it in older code and then forgot to remove it. Thanks.